utils/src/ooinstall/cli_installer.py

```diff
@@ -72,7 +72,7 @@ def delete_hosts(hosts):
                 click.echo("\"{}\" doesn't coorespond to any valid input.".format(del_idx))
     return hosts, None
 
-def collect_hosts(oo_cfg, masters_set=False, print_summary=True):
+def collect_hosts(oo_cfg, existing_env=False, masters_set=False, print_summary=True):
```

Comment:
I should add that once we have a more intelligent callback plugin, then we could add additional states (or substates) based on where a failure happens to make decisions on whether or not a re-run should even be attempted.